getting-started/exploring-enterprise.md

```diff
@@ -17,7 +17,8 @@ To upgrade or obtain a full Enterprise license key, please [contact sales][conta
 ## Activating TimescaleDB Enterprise
 
 Before you can activate TimescaleDB Enterprise, you must have a Timescale-Licensed version of 
-TimescaleDB up and running. Note that our standard binary releases are by default Timescale-Licensed. 
+TimescaleDB up and running. Note that our standard binary releases contain code that is licensed under 
+both Apache License Version 2.0 and the Timescale License. 
```

Comment:
Not clear whether some binary releases have code that is _all_ under Apache 2 and some releases have code _all_ under TSL, OR whether standard binary releases have _some_ code that is Apache 2 and _some_ that is TSL.

Reply:
All default binary releases will have some code under Apache 2 and some under TSL. I'm thinking maybe just remove that note? Delete starting from "Note that"?

Reply:
Fixing